folio-token/pom.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.folio</groupId>
+    <artifactId>folio-spring-support</artifactId>
+    <version>7.2.0-SNAPSHOT</version>
+  </parent>
+
+  <name>folio-token</name>
+  <artifactId>folio-token</artifactId>
+  <description/>
+
+  <properties>
+    <folio-spring-base.version>6.0.1</folio-spring-base.version>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
+    <edge-api-utils-version>1.2.0</edge-api-utils-version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>folio-spring-base</artifactId>
+      <version>${folio-spring-base.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.kafka</groupId>
+      <artifactId>spring-kafka</artifactId>
```

Comment:
How kafka is related to token or system user?

Reply:
@nirmalr-shah I agree with all of these changes.

Reply:
removed